frontend/src/core/wasm/bridge.ts

```diff
@@ -514,6 +514,11 @@ export class PyodideBridge implements RunRequests, EditRequests {
     return response;
   };
 
+  getDependencyTree = async () => {
+    // WASM doesn't support dependency trees yet
+    return { tree: undefined };
```

Comment:
Just wondering what the limitation is. Because micropip vs uv?

Reply:
the tree is parsed from "uv tree"